### PR TITLE
/describe endpoint: meta, parameter_meta, images

### PR DESCRIPTION
**Pull request by aednichols**
Addresses most of https://github.com/broadinstitute/cromwell/issues/4574: `meta`, `parameter_meta`, and `images`.

`importedDescriptorTypes` requires some more thought and is not urgently needed yet, so it'll be in a subsequent PR.

### CODE REVIEW

File: engine/src/test/scala/cromwell/webservice/routes/CromwellApiServiceSpec.scala [hunk 1]
```diff
@@ -617,7 +617,7 @@ object CromwellApiServiceSpec {
               s"[reading back DescribeRequest contents] version: ${sourceFiles.workflowTypeVersion}"
             )
 
-            sender ! DescribeSuccess(description = WorkflowDescription(valid = true, readBack))
+            sender ! DescribeSuccess(description = WorkflowDescription.empty.copy(valid = true, errors = readBack))
```
Comment (mcovarr) [inline engine/src/test/scala/cromwell/webservice/routes/CromwellApiServiceSpec.scala]: TOL this feels odd. Perhaps a companion object `apply` that sets the defaults instead??

File: services/src/test/resources/describe/wdl_1_0/name_from_task/description.json [hunk 2]
```diff
@@ -16,6 +33,11 @@
...
   "meta" : {
+    "email" : "skroob@spaceballs.gov",
+    "author" : "President Skroob",
+    "description" : "Spaceballs: The Unit Test"
+  },
+  "parameterMeta" : {
 
   }
-}
+}
```
Comment (mcovarr) [inline services/src/test/resources/describe/wdl_1_0/name_from_task/description.json]: \n

File: services/src/test/resources/describe/wdl_1_0/name_from_workflow/description.json [hunk 3]
```diff
@@ -17,5 +19,8 @@
   ],
   "meta" : {
 
+  },
+  "parameterMeta" : {
+
   }
-}
+}
```
Comment (mcovarr) [inline services/src/test/resources/describe/wdl_1_0/name_from_workflow/description.json]: \n

File: services/src/test/resources/describe/wdl_1_0/name_none_from_tasks/description.json [hunk 4]
```diff
@@ -17,5 +19,8 @@
   ],
   "meta" : {
 
+  },
+  "parameterMeta" : {
+
   }
-}
+}
```
Comment (mcovarr) [inline services/src/test/resources/describe/wdl_1_0/name_none_from_tasks/description.json]: \n

File: services/src/test/resources/describe/wdl_draft_2/name_from_task/description.json [hunk 5]
```diff
@@ -16,6 +33,11 @@
...
   "meta" : {
+    "email" : "skroob@spaceballs.gov",
+    "author" : "President Skroob",
+    "description" : "Spaceballs: The Unit Test"
+  },
+  "parameterMeta" : {
 
   }
-}
+}
```
Comment (mcovarr) [inline services/src/test/resources/describe/wdl_draft_2/name_from_task/description.json]: \n

File: services/src/test/resources/describe/wdl_draft_2/name_from_workflow/description.json [hunk 6]
```diff
@@ -17,5 +19,8 @@
   ],
   "meta" : {
 
+  },
+  "parameterMeta" : {
+
   }
-}
+}
```
Comment (mcovarr) [inline services/src/test/resources/describe/wdl_draft_2/name_from_workflow/description.json]: \n

File: wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/Util.scala [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    val metaMap = meta.map(_.meta).getOrElse(Map.empty)
+    val parameterMetaMap = parameterMeta.map(_.metaAttributes).getOrElse(Map.empty)
+
+    (stringifyMetaValues(metaMap), stringifyMetaValues(parameterMetaMap))
+  }
+
+
+
```
Comment (mcovarr) [inline wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/Util.scala]: lotta whitespace here

File: engine/src/test/scala/cromwell/webservice/routes/CromwellApiServiceSpec.scala (hunk 1 above)
Comment (aednichols) [reply engine/src/test/scala/cromwell/webservice/routes/CromwellApiServiceSpec.scala]: I somehow forgot about `apply` and insisted on reinventing it myself

**Pull request merged**